docker-compose.yml

```diff
@@ -129,20 +116,22 @@ services:
     volumes:
       - icingaweb:/data
 
-  icingaweb-mysql:
+  mysql:
+    # see https://hub.docker.com/_/mariadb
+    image: mariadb
     command: --default-authentication-plugin=mysql_native_password
     environment:
       MYSQL_RANDOM_ROOT_PASSWORD: 1
-      MYSQL_DATABASE: icingaweb
-      MYSQL_USER: icingaweb
-      MYSQL_PASSWORD: ${ICINGAWEB_MYSQL_PASSWORD:-icingaweb}
-    image: mysql
+      ICINGAWEB_MYSQL_PASSWORD: ${ICINGAWEB_MYSQL_PASSWORD}
+      ICINGADB_MYSQL_PASSWORD: ${ICINGADB_MYSQL_PASSWORD}
     logging: *default-logging
     volumes:
-      - icingaweb-mysql:/var/lib/mysql
+      # - ./mysql:/var/lib/mysql
+      - mysql:/var/lib/mysql
+      - ./init-mysql.sh:/docker-entrypoint-initdb.d/init-mysql.sh
```

Comment:
Please move `init-mysql.sh` to `mysql` (rename it from `icingadb-mysql` and remove `privileges.sql`)  and mount `./env/mysql:/docker-entrypoint-initdb.d`.

Reply:
- [x] done